test/entrypoint/config.yml

```diff
@@ -0,0 +1,71 @@
+credentials:
+  users:
+    guest:
+      roles: [super]
+    storage:
+      roles: [sharding]
+      password: "storage"
+
+iproto:
+  listen:
+    - uri: 'unix/:./{{ instance_name }}.iproto'
+  advertise:
+    sharding:
+      login: 'storage'
+      password: 'storage'
+
+sharding:
+  bucket_count: 1234
+  sched_ref_quota: 258
+
+groups:
+  group-001:
+    replicasets:
+      replicaset-001:
+        sharding:
+          roles: [router]
+        roles:
+          - roles.sharded-queue-router
+        database:
+          replicaset_uuid: 'aaaaaaaa-0000-4000-b000-000000000000'
+        instances:
+          queue-router:
+            database:
+              instance_uuid: 'aaaaaaaa-aaaa-4000-b000-000000000001'
+              mode: rw
```

Comment:
master-master replicaset of routers?

Reply:
But replicaset of two router is still a bit weird. Or does sharded queue routers stateful?

Reply:
Oh, I just copy-pasted the configuration from the test configuration for Cartridge's cluster (there is some tests with a second router).

Yes, it could be two routers in two different replicasets.